tests/utils/test_validation.go

```diff
@@ -53,22 +53,26 @@ func VerifySimpleJsonScanResults(t *testing.T, content string, minVulnerabilitie
 	}
 }
 
-func VerifySimpleJsonJasResults(t *testing.T, content string, minSastViolations, minIacViolations, minSecrets, minApplicable int) {
+func VerifySimpleJsonJasResults(t *testing.T, content string, minSastViolations, minIacViolations, minSecrets, minApplicable, minNotCovered int) {
```

Comment:
notApplicable is always 1? why not pass it as a parameter as well? what about undetermined status?